language-support/ts/codegen/src/DA/Daml/Daml2Ts.hs

```diff
@@ -0,0 +1,18 @@
+-- Copyright (c) 2020 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+{-# LANGUAGE DerivingStrategies #-}
+module DA.Daml.Daml2Ts (
+    NpmPackageName(..)
+  , NpmPackageVersion(..)
+  ) where
+
+import qualified Data.Text.Extended as T
+import Data.Hashable
+import Data.Aeson
+
+newtype NpmPackageName = NpmPackageName {unNpmPackageName :: T.Text}
```

Comment:
Why do we have this twice now?

Reply:
We use it to configure constants in `TsCodeGenMain.hs` and later to configure (different constants) in `Daml2Ts.hs` (lint test).

Reply:
To clarify, this change means we have a single definition now where there were two.

Reply:
No, you still have two definitions, one here and one in `DA.Daml.Daml2TsUtils` which seems confusing given that the whole point of this PR seems to be to factor this out.

Reply:
Ah! Sorry, a mistaken commit. Fixed now.